test/hotspot/jtreg/compiler/codecache/CheckCodeCacheInfo.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2022, Amazon.com Inc. or its affiliates. All rights reserved.
```

Comment:
We don't specify a year any more.
The correct one is "Copyright Amazon.com Inc. or its affiliates. All Rights Reserved." See https://github.com/openjdk/jdk/pull/6915